### ISSUE TITLE
Possibility to set language 

### ISSUE THREAD

**Issue by salgo60** [yarl/gravemap#2]
Graves in Stockholm has swedish labels see http://graves.wiki/#/map/@59.304109,17.864799,11z

Would be nice to set lang = sv ==>

SERVICE wikibase:label { bd:serviceParam wikibase:language "sv". }

EDIT: I have now set the en language labels so it looks ok but feels like local language is prefered....

**Question any plans of the future?!?!?**
I have done a rather lot of [odd searches on the Swedish cemetery in Stockholm](http://minancestry.blogspot.se/2016/12/norra-begravningsplatsen.html)
* [Birth places of persons on a cemetery](http://minancestry.blogspot.se/2016/12/birth-places-of-people-buried-on.html)
* [Timeline of the people on the cemetery](https://goo.gl/u49ZuX)
* [graves were the person in the grave is also available on Spotify](https://goo.gl/WgaH5v)

**Comment by yarl** [comment]
@salgo60 yes, it's planned! I will probably implement that during December.

Sorry, somehow I unwatched this repo on GitHub and had no notifications about issues.

**Comment by dubyk** [comment]
Hi!
I also support the request for adding a local languages (Ukrainian, Polish, Russian is interested).

**Comment by pyrog** [comment]
You could select automatically user language, and by default english.
